frontend/test/__support__/ui.tsx

```diff
@@ -178,7 +178,14 @@ export function queryIcon(name: string, role: ByRoleMatcher = "img") {
  * screen.getByText(getBrokenUpTextMatcher("my text with a styled word"))
  */
 export function getBrokenUpTextMatcher(textToFind: string): MatcherFunction {
-  return (content, element) => element?.textContent === textToFind;
+  return (content, element) => {
+    const hasText = (node: Element | null | undefined) =>
+      node?.textContent === textToFind;
+    const childrenDontHaveText = element
```

Comment:
nit: we don't have the use of `dont` in FE code, but we have a few `doNot` references. This change is for consistency.
```suggestion
    const childrenDoNotHaveText = element
```